app/javascript/components/pages/EventShow.js

```diff
@@ -9,7 +43,12 @@ class EventShow extends Component {
       location,
       user_id,
     } = this.props.party;
-    const filteredItems = this.props.items.filter((item) => {
+
+    if(this.state.event === false){
```

Comment:
```suggestion
    if(!this.state.event){
```
we can use the truthiness of the event. If event is undefined it will early return